input/pagecontent/sushi.md

```diff
@@ -148,10 +153,11 @@ A populated **ig-data** directory should look something like this:
         └── 3_myFourthPage.md
 ```
 
-Populate these directories as follows:
+Populate your project as follows:
 
+* **package.json**: This required file is the package manifest. The content is described [here](https://confluence.hl7.org/pages/viewpage.action?pageId=35718629#NPMPackageSpecification-Packagemanifest). 
+* **package-list.json**: This optional file should contain the version history of your IG. If present, it will be used instead of a generated **package-list.json**.
```

Comment:
Consider referencing the package-list documentation here: https://confluence.hl7.org/display/FHIR/FHIR+IG+PackageList+doco

Reply:
done